src/System.Runtime.Serialization.Formatters/tests/BinaryFormatterTestData.cs

```diff
@@ -0,0 +1,368 @@
+﻿using System.Collections.Generic;
+using System.Data;
+using System.Data.SqlTypes;
+using System.Globalization;
+using System.Linq;
+using System.Net;
+using System.Numerics;
+using System.Text;
+
+namespace System.Runtime.Serialization.Formatters.Tests
+{
+    public partial class BinaryFormatterTests
+    {
+        /// <summary>
+        /// First: Object
+        /// Second: Core Base64 serialized string
+        /// Third: Netfx Base64 serialized string 
```

Comment:
At some point in the future we'll have to update these base64 values. Eg we added a field in hopefully a non breaking way. At that point we will need to have blobs for both old and new and test both directions. I guess that is a problem to solve at that time.